src/Service/EolValidator.php

```diff
@@ -87,7 +87,13 @@ public function validateServiceEol(): array
             $service = $this->serviceFactory->create($serviceName);
             $serviceVersion = $service->getVersion();
 
-            if ($validationResult = $this->validateService($serviceName, $serviceVersion)) {
+            /**
+             * Hack to validate the mariadb version; getting the version from relationships returns 'mysql:<version>'.
```

Comment:
Let's not call this hack. Just describe the reason

Reply:
Explaining code with comments is not a good practice. Better to add this in description to the method if really needed.

Reply:
You can move this condition `$serviceName == 'mysql' ? 'mariadb' : $serviceName` to a new method like
```
function convertServiceNames($serviceName) {
    return $serviceName == 'mysql' ? 'mariadb' : $serviceName;
}
```
And add description to the method.